lighthouse-core/config/default.js

```diff
@@ -604,42 +607,48 @@ module.exports = {
     },
     "a11y-well-structured": {
       "title": "Elements Are Well Structured",
-      "description": "Screen readers and other assitive technologies require annotations to understand otherwise ambiguous content."
+      "description": "Screen readers and other assistive technologies require annotations to understand otherwise ambiguous content."
     },
     "a11y-aria": {
       "title": "ARIA Attributes Follow Best Practices",
-      "description": "Screen readers and other assitive technologies require annotations to understand otherwise ambiguous content."
+      "description": "Screen readers and other assistive technologies require annotations to understand otherwise ambiguous content."
     },
     "a11y-correct-attributes": {
       "title": "Elements Use Attributes Correctly",
-      "description": "Screen readers and other assitive technologies require annotations to understand otherwise ambiguous content."
+      "description": "Screen readers and other assistive technologies require annotations to understand otherwise ambiguous content."
     },
     "a11y-element-names": {
       "title": "Elements Have Discernable Names",
-      "description": "Screen readers and other assitive technologies require annotations to understand otherwise ambiguous content."
+      "description": "Screen readers and other assistive technologies require annotations to understand otherwise ambiguous content."
     },
     "a11y-language": {
       "title": "Page Specifies Valid Language",
-      "description": "Screen readers and other assitive technologies require annotations to understand otherwise ambiguous content."
+      "description": "Screen readers and other assistive technologies require annotations to understand otherwise ambiguous content."
     },
     "a11y-meta": {
       "title": "Meta Tags Used Properly",
-      "description": "Screen readers and other assitive technologies require annotations to understand otherwise ambiguous content."
+      "description": "Screen readers and other assistive technologies require annotations to understand otherwise ambiguous content."
+    },
+    "manual-pwa-checks": {
+      "title": "¹ Manual checks to verify",
+      "description": "These audits are required by the baseline " +
+          "[PWA Checklist](https://developers.google.com/web/progressive-web-apps/checklist) but are " +
+          "not automatically checked by Lighthouse. They do not affect your score but please verify them manually."
     },
   },
   "categories": {
     "pwa": {
       "name": "Progressive Web App",
       "weight": 1,
-      "description": "These audits validate the aspects of a Progressive Web App. They are a subset of the [PWA Checklist](https://developers.google.com/web/progressive-web-apps/checklist).",
+      "description": "These audits validate the aspects of a Progressive Web App. They are a subset¹ of the baseline [PWA Checklist](https://developers.google.com/web/progressive-web-apps/checklist).",
```

Comment:
👍  this also gets rid of the strange looking 1 that doesn't really look like a footnote at the beginning of the group :)